lms/djangoapps/certificates/management/commands/modify_cert_template.py

```diff
@@ -0,0 +1,91 @@
+"""Management command to modify certificate templates."""
+import logging
+import shlex
+from argparse import RawDescriptionHelpFormatter
+
+from django.core.management.base import BaseCommand, CommandError
+
+from lms.djangoapps.certificates.models import (
+    ModifiedCertificateTemplateCommandConfiguration,
+)
+from lms.djangoapps.certificates.tasks import handle_modify_cert_template
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """Management command to modify certificate templates.
+    Example usage:
+    ./manage.py lms modify_cert_template --old-text
```

Comment:
The example seems incomplete at a first glance. Should this also have an example of `--old-text` and how to use the other options (like `--new_textc {blah}`)?